plugin/src/main/java/org/opensearch/sql/plugin/catalog/CatalogServiceImpl.java

```diff
@@ -117,16 +120,12 @@ private <T> T doPrivileged(PrivilegedExceptionAction<T> action) {
   }
 
   private StorageEngine createStorageEngine(CatalogMetadata catalog) throws URISyntaxException {
-    StorageEngine storageEngine;
     ConnectorType connector = catalog.getConnector();
     switch (connector) {
```

Comment:
switch is not necessary?
```
connectorTypeStorageEngineFactoryMap.get(connector).getStorageEngine(catalog.getName(), catalog.getProperties());
```

Reply:
In case if someone provides opensearch as connector. It will throw NullPointerException without a proper message.